docs/user-manual/civiform-admin-guide/manage-questions.md

```diff
@@ -33,6 +33,21 @@ Input: 123-45-6789 Example obfuscated output: d158596dd5a6cae6fcb282832885631553
 
 Input: 123-45-6780 Example obfuscated output: 01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b
 
+### Universal questions
+
+Universal questions are questions that are intended to be used by all programs. When a question is marked as universal, it appears at the top in a separate section in the questions tab and the question bank. Any question may be set as a universal question.
```

Comment:
suggestion (non-blocking):  "question bank" is used in a more abstract way higher up in the page under the "Create a question" header.  I think here you're referring to the question list sidebar that appears when adding a question to a program, right?  Could we clarify by adding a screenshot of that too?  I might also refer to it as the "question list sidebar" rather than re-using the term "question bank", but I recognize that "question list sidebar" isn't completely clear either.